lib/ansible/modules/system/kernel_blacklist.py

```diff
@@ -129,6 +129,9 @@ def main():
     if module.params['blacklist_file']:
         filename = module.params['blacklist_file']
 
+    if module.check_mode:
+        filename = '/dev/null'
```

Comment:
Using `os.devnull` might be better.

https://docs.python.org/2/library/os.html#os.devnull